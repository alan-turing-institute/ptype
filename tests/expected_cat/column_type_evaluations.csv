,ptype-cat
overall-accuracy,0.13
boolean,
date,
float,0.08
integer,0.07
string,