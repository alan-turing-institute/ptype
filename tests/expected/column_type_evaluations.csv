,ptype
overall-accuracy,0.93
boolean,0.65
date,0.62
float,0.92
integer,0.87
string,0.93
